doc/development/rfc/version_numbering.md

```diff
@@ -0,0 +1,161 @@
+# RFC: Version Numbering
+
+Author: Vaclav Petras
+
+Status: Draft
+
+## Summary
+
+Use version number in a format _major.minor.micro_ where
+- _major_ is for large additions or backwards incompatible changes,
+- _minor_ is for additions, and
+- _micro_ is for fixes.
+These numbers are assigned to releases. Development versions, pre-release,
+and builds are marked by additional labels as suffixes.
+All major and minor version numbers are used for releases,
+i.e., there is no distinction between even and odd numbers.
+
+## Background
+
+The increasing versions was always following the same ideas as in
+the Semantic Versioning, i.e., _major_, _minor_, and _patch_
```

Comment:
```suggestion
the [Semantic Versioning](https://semver.org/), i.e., _major_, _minor_, and _patch_
```

Link added as a guide for newcomers

Reply:
Done for the first mention of Semantic Versioning. It is also linked in the related documents section.

```
The increasing versions was always following the same ideas as in
the [-Semantic Versioning,-]{+[Semantic Versioning](https://semver.org/),+}
i.e., _major_, _minor_, and [-_patch_-]{+_patch_,+}
with major for breaking changes, minor for backwards compatible
features, and patch for fixes.
```